bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/ParallacticAngleControls.scala

```diff
@@ -151,6 +160,7 @@ class ParallacticAngleControls(isPaUi: Boolean) extends GridBagPanel with Publis
       val spObs = ispObs.getDataObject.asInstanceOf[SPObservation]
       spObs.setSchedulingBlock(ImOption.apply(sb))
       ispObs.setDataObject(spObs)
+      Swing.onEDT(callback.run())
       resetComponents()
```

Comment:
Yeah `updateSchedulingBlock` is only called on the EDT s I should just remove `Swing.onEDT` because it's confusing.